core/windows_and_tabs/windows_and_tabs_linux.py

```diff
@@ -38,3 +38,6 @@ def window_hide_others():
 
     def window_open():
         actions.key("ctrl-n")
+
+    def focus_last():
```

Comment:
One subtlety with actions is that they need to be declared somewhere first. The app actions are owned by talon and can't be extended. 
 
I would suggest declaring this in app_switcher.py

    def switcher_focus_last():
        """Focus last window/application"

Then for each of the supported operating systems you just need to add the appropriate implementation.  eg the below for [windows_and_tabs_mac.py](https://github.com/talonhub/community/pull/1336/files#diff-91df1befe13e34f8155ad02af92009a9900e4f2d1734ca458d13b2ae536fcaf8)

@ctx.action_class("user")
class UserActions:
    def switcher_focus_last():
        actions.key("cmd-tab")

Reply:
Okay-how does it look now?